nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -153,11 +223,24 @@ Nav2Panel::Nav2Panel(QWidget * parent)
   accumulated_wp_->assignProperty(navigation_mode_button_, "enabled", false);
   accumulated_wp_->assignProperty(navigation_mode_button_, "toolTip", waypoint_goal_msg);
 
+  accumulated_wp_->assignProperty(save_waypoints_button_, "text", "Save WPs");
+  accumulated_wp_->assignProperty(save_waypoints_button_, "enabled", false);
+
+  accumulated_wp_->assignProperty(load_waypoints_button_, "text", "Load WPs");
+  accumulated_wp_->assignProperty(load_waypoints_button_, "enabled", false);
+
+  accumulated_wp_->assignProperty(pause_waypoint_button_, "text", "Pause WP");
+  accumulated_wp_->assignProperty(pause_waypoint_button_, "enabled", true);
+
+  accumulated_wp_->assignProperty(nr_of_loops_, "enabled", false);
+  accumulated_wp_->assignProperty(store_initial_pose_checkbox_, "enabled", false);
+
   accumulated_nav_through_poses_ = new QState();
   accumulated_nav_through_poses_->setObjectName("accumulated_nav_through_poses");
   accumulated_nav_through_poses_->assignProperty(start_reset_button_, "text", "Cancel");
   accumulated_nav_through_poses_->assignProperty(start_reset_button_, "toolTip", cancel_msg);
   accumulated_nav_through_poses_->assignProperty(start_reset_button_, "enabled", true);
+  accumulated_nav_through_poses_->assignProperty(store_initial_pose_checkbox_, "enabled", false);
```

Comment:
Should this state have some specification for the Save/Load waypoints buttons? I would think they should be disabled.

Reply:
Yes "enabled" but false .. which is disabled.. seems bit funny though..

Reply:
I don't see any reference to `save_waypoints_button_` or `load_waypoints_button_` under the `accumulated_nav_through_poses_` state

Reply:
Sorry got confused! 

This needs to be implemented..